glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -130,13 +131,33 @@ open class GleanInternalAPI internal constructor () {
             maxEvents = this.configuration.maxEvents
         )
 
-        handle = LibGleanFFI.INSTANCE.glean_initialize(cfg)
+        // Start the migration from glean-ac, if needed.
+        val migrator = GleanACDataMigrator(applicationContext)
+
+        // TODO: like we do in glean-ac, we should check if we already have a client id.
+        // If we don't, then we don't need to go through migration and this is likely a new
+        // client.
+        val newSequenceNums = if (migrator.shouldMigrateData()) {
```

Comment:
Can we wrap the `else` branch in `{}`? I find it being inline a bit hard to read.